op-node/p2p/gossip.go

```diff
@@ -880,6 +1016,7 @@ type GossipOut interface {
 	PublishL2RequestResponse(ctx context.Context, msg *eth.ExecutionPayloadEnvelope, signer Signer) error
 	PublishL2Request(ctx context.Context, hash common.Hash) error            // TODO: add signer, sign request
 	PublishL2EndOfSequencingRequest(ctx context.Context, epoch uint64) error // TODO: add signer, sign request
+	PublishPreconfirmation(ctx context.Context, sc SignedCommitment) error
```

Comment:
PublishPreconfirmationCommitment or PublishSignedCommitment (publishL2Payload is also a preconfirmation)